src/main/api/2/studio-api-2.yaml

```diff
@@ -3486,6 +3486,92 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content/paste:
+    post:
+      tags:
+        - content
+      summary: Copy or cut the given items to the target path
```

Comment:
Paste a tree of items to a target path (copy content or move content). To be used by the UI in the clipboard implementation.